Mono.Addins/Mono.Addins/AddinManager.cs

```diff
@@ -828,7 +828,7 @@ public static void RemoveExtensionNodeHandler (Type instanceType, ExtensionNodeE
 			remove { AddinEngine.AddinAssembliesLoaded -= value; }
 		}
 		
-		internal static bool CheckAssembliesLoaded (HashSet<string> files)
+		public static bool CheckAssembliesLoaded (HashSet<string> files)
```

Comment:
Why are those methods being made public?

Reply:
In Mono.Addins.Setup/AddinPackage.cs:257 requires that function ?
```
if (AddinManager.CheckAssembliesLoaded (files) || files.Any (f => HasWriteLock (f))) {
```

Reply:
The AssemblyInfo files that are marked with `<Compile Remove="AssemblyInfo.cs" />` were handling InternalsVisibleTo from `Mono.Addins` to `Mono.Addins.Setup`.

https://github.com/mono/mono-addins/blob/4ae477426811bfd913e10d6940a5eec43ab07ae3/Mono.Addins/AssemblyInfo.cs#L22-L29

See above comments.

Reply:
Has added InternalsVisibleTo to the project file and revert back to internal